### DISCUSSION TITLE
W3C Community Group Proposal and Second Meeting Scheduling

### DISCUSSION THREAD

**Discussion by robertlong** [omigroup/omigroup#11] [General]
Hello everyone! Sunday's inaugural meeting was a huge success! We had over 35 attendees! I did not expect this level of interest this early, so thank you to everyone who showed up!

## Summary of the last meeting:

The meeting focused on getting to know each other, trying to define the "Open Metaverse", and figuring out how we want to organize ourselves.

There's a recording available here: 
https://www.youtube.com/watch?v=x3mFHxv9CNQ

And @madjin took notes which are available here:
https://hackmd.io/@XR/omi-1

## Action items from the meeting:

- Establish a GitHub organization for discussion and spec work (This org)
- Set up a Matrix room and bridges to existing Discord communities (I'm still working on this)
- Further investigate how we should organize ourselves and how we can work with/within existing organizations (W3C, DIF, Khronos, Matrix, etc.)
  - Gather information about W3C Community Groups
  - Determine if we can work with or be a part of an existing group 
  - Determine options for IP Protection
- Organize and present the data from the OMI Survey
- Schedule the next meeting based on the Doodle survey results

## Next Steps

Over the last week I've been speaking with different people involved in various W3C groups including @cwilso, @TrevorFSmith, @humbletim, and @dmitrizagidulin. I also attended the W3C Credentials Community Group's weekly meeting to get a sense for what they are working on there.

I received multiple recommendations for us to start our our own W3C Community Group. I think this is likely the best option for us. I did talk to @TrevorFSmith and @cwilso about possibly joining up with the W3C Immersive Web Community Group, but they are focused on lower-level browser API specifications such as WebXR, lighting estimation, hand tracking, etc. There is likely a lot that we can collaborate on, but it would be best for us to do this work in our own group with a sole focus on metaverse-related specifications.

You can read more about W3C Community Groups [here](https://www.w3.org/community/about/faq/).

I'd like to propose that we create a W3C Metaverse Interoperability Community Group for the purposes of developing specifications, holding discussions, and developing reference implementations.

The first step of this process is proposing a group on the [W3C website](https://www.w3.org/community/groups/propose_cg/):

![image](https://user-images.githubusercontent.com/1753624/114951721-d4fa4d80-9e09-11eb-8901-bcd5fccf1232.png)

We will need to decide on a name, description, and short name for the group.

Five individuals will then need to provide their support to the proposal by creating a W3C account and voting on their site. Pending W3C's approval we will then get a launch date for our group when we will be given access to the W3C Community Group infrastructure.

We will also need to nominate and vote on one or more community group chairs. These individuals will be responsible for making sure the group fulfills the [W3C's community group requirements](https://www.w3.org/community/about/process/#operational-agreements) as well as any of our group's operational agreements which we will also need to determine. There's a process for chair selection documented [here](https://w3c.github.io/cg-charter/CGCharter.html#chairs), but ultimately it's up to us to choose our own chairs.

Contributors to specifications will need to sign the [W3C Community Contributor License Agreement (CLA)](https://www.w3.org/community/about/process/cla/). This grants W3C royalty-free license to your individual contributions. You can read more about the IP protections [here](https://www.w3.org/community/about/process/#patents).

You also cannot participate in a community group anonymously. You must have an identity in the community and follow the [W3C Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/). That also applies to all of our communications channels.

So, with all of that said. What will the activities of this group look like? We will propose and discuss specification work in this Github repository via Github issues. Specifications will get their own repository. Long form discussion, Q&A, and other topics dealing with public administrative work can happen in Github Discussions. We also will have both a public/private mailing list provided to us by W3C. I'd personally like to use these sparingly beyond meeting agendas, notes, and reminders. I'd opt for using github discussions and issues instead. Realtime communication will happen in our Matrix room, which I am still in the process of setting up due to our unique bridging requirements. Until then, we will continue meeting in the [AngellXR Discord](https://discord.gg/bnkTMCaRQe).

Specification and implementation repositories are something we can discuss in detail later. I still need some guidance on this, but I'll wait until we've decided on this group or not before we continue.

tldr; A W3C Community Group gives us a formal spec process, credibility when talking to larger companies, an IP protection framework that's already well understood and accepted by companies, and a bunch of resources that we can use.

I'd like to talk more about this here asynchronously so that everyone gets a chance to participate. I'd also like to schedule our next meeting where we can wrap up these discussions on the call.

Based off the Doodle Survey results, it looks like people are most available in the following time slots:
- Sundays 10:00 AM - 11:00 AM Pacific Time
- Sundays 11:00 AM - 12:00 PM Pacific Time
- Mondays 11:00 AM - 12:00 PM Pacific Time
- Mondays 12:00 PM - 1:00 PM Pacific Time
- Mondays 4:00 PM - 5:00 PM Pacific Time
- Mondays 5:00 PM - 6:00 PM Pacific Time
- Wednesdays 8:00 AM - 9:00 AM Pacific Time
- Wednesdays 10:00 AM - 11:00 AM Pacific Time
- Wednesdays 4:00 PM - 5:00 PM Pacific Time
- Thursdays 4:00 PM - 5:00 PM Pacific Time
- Thursdays 5:00 PM - 6:00 PM Pacific Time
- Thursdays 6:00 PM - 7:00 PM Pacific Time
- Fridays 8:00 AM - 9:00 AM Pacific Time
- Fridays 12:00 PM - 1:00 PM Pacific Time
- Fridays 1:00 PM - 2:00 PM Pacific Time
- Fridays 2:00 PM - 3:00 PM Pacific Time
- Saturdays 11:00 AM - 12:00 PM Pacific Time
- Saturdays 1:00 PM - 2:00 PM Pacific Time

We had 17 participants in that poll and it didn't narrow down the results enough so I'm going to create another with just these times.

# You can vote for the timeslots [here](https://doodle.com/poll/xt9b9t9zpwg7qkr9)

So now let's discuss this W3C Working Group Proposal, any objections, and the details for the proposal (name, shortname, description).

Thanks again for getting involved everyone!

**Comment by yangwao** [comment]
AngellXR Discord invite seems invalid )

**Reply by robertlong** [reply]
Thanks, it should work now.

**Comment by robertlong** [comment]
I'm about ready to close the availability poll for the next meeting. Monday 8:00 AM - 9:00 AM Pacific Time has the most votes, but I'm probably not going to be able to do that this week. I've got my second vaccine appointment the night before and I know the side effects can be rough.

![image](https://user-images.githubusercontent.com/1753624/115134544-f6a03400-9fc5-11eb-895b-23a25453dcdd.png)

I'd say we should shoot for one of the time slots with 8 votes. I'd like to propose we meet on Friday 12:00 PM - 1:00 PM PDT. That'll give us ample time to get the word out. I also think we can better accomplish things asynchronously here before the next meeting.

Our agenda for Friday will be to take a vote on forming this W3C Community Group, we also need to schedule a date and time for the [OKR workshop](https://github.com/omigroup/OMI/discussions/10).

Here's what we need:

- If you are in opposition to forming this W3C Community Group, please state your reasons and any alternatives.
- For the W3C Community Group Application we need:
  - A group name
  - A short group name used in urls/mailing lists
  - A description of the group that briefly explains our scope of work.
    - For example the Immersive Web Community Group's is:
      > Our goal is to help bring high-performance Virtual Reality and Augmented Reality to the open Web.
- Proposals for additional meeting agenda items. This is going to be a 1 hour meeting so please be conscious of the time. We'll try to get though as many things as we can.

Here's my proposal:

#### Group Name:
Metaverse Interoperability Group
#### Short Name:
metaverse-interop
#### Description:
Our mission is to bridge virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more.

**Comment by robertlong** [comment]
Ok the final time for the meeting will be Friday, April 23rd 12:00 PM - 1:00 PM PDT.

We'll be meeting in Discord voice in the [AngellXR server](https://discord.gg/JRC8CvBUwR).

**Comment by yangwao** [comment]
ooo, noted in calendar!
May show up as it's 9 pm CEST
Thank you for organizing this!
